build.gradle

```diff
@@ -439,7 +436,7 @@ subprojects {
     }
     referenceTest {
       java {
-        srcDir file('src/referenceTest/generated')
+        srcDir file('src/referenceTest/generated_tests')
```

Comment:
This makes IntelliJ a lot happier by not having production code in a test source tree.